package.json

```diff
@@ -12,6 +12,11 @@
   "license": "ISC",
   "dependencies": {
     "express": "^4.15.3",
+    "geckodriver": "^1.8.0",
+    "istanbul": "^0.4.5",
```

Comment:
We don't need this dependency - we're using nyc for coverage - assume this dependency got copy pasted some how from the Mocha testing I did?